taxonomies/Message.ttl

```diff
@@ -385,6 +385,51 @@ ids:LogMessage a owl:Class;
     rdfs:label "Log Message"@en ;
     rdfs:comment "Log Message which can be used to transfer logs e.g., to the clearing house."@en.
 
+ids:LogEntry a owl:Class;
+    rdfs:label "Log Entry"@en ;
+    rdfs:comment "Contains the metadata of a LogMessage"@en .
+
+ids:transactionID a owl:DatatypeProperty ;
+    rdfs:domain ids:LogEntry ;
+    rdfs:range xsd:unsignedInt ;
+    rdfs:comment "Links a LogEntry (Metadata) to a transactionID"@en .
+
+ids:processID a owl:DatatypeProperty ;
+    rdfs:domain ids:LogEntry ;
+    rdfs:range xsd:string ;
+    rdfs:comment "Links a LogEntry (Metadata) to a processID"@en .
+
+ids:documentID a owl:DatatypeProperty ;
+    rdfs:domain ids:LogEntry ;
+    rdfs:range xsd:string ; # UUID?
+    rdfs:comment "Links a LogEntry (Metadata) to a documentID"@en .
+
+# Timestamp given by dct:created
+
+ids:hasHash a owl:DatatypeProperty ;
```

Comment:
Be more consistent with naming conventions? The other new properties do not have a verb notation (clientID, payload, signature, ...), but this one has (**has**Hash). Easy solution = remove the "has" part from this line?

```suggestion
ids:hash a owl:DatatypeProperty ;
```